### ISSUE TITLE
Possible to pull in calendar attendee email/name with single request to FindItems?

### ISSUE THREAD

**Issue by kellenkm** [jamesiarmes/php-ews#274]
I'm trying to figure out how to pull in attendee info (event.RequiredAttendees and event.OptionalAttendees) in a single request (DisplayTo doesn't always have attendee names / emails). I have it setup pulling in the info using separate requests for each event after the initial request (similar to the solution [here][https://github.com/jamesiarmes/php-ews/issues/104]), but it takes a long time (1 minute or more), since I'm pulling in a lot of calendar data. 

It looks like the EWS API supports pulling in additional info in a single request using PropertySets:
http://blogs.msdn.com/b/exchangedev/archive/2010/03/16/loading-properties-for-multiple-items-with-one-call-to-exchange-web-services.aspx

I tried this, but got an error saying the OptionalAttendees attribute wasn't supported for this type of request:

		$request = new EWSType_FindItemType();

		$request->Traversal = EWSType_ItemQueryTraversalType::SHALLOW;
		$request->ItemShape = new EWSType_ItemResponseShapeType();
		$request->ItemShape->BaseShape = EWSType_DefaultShapeNamesType::ALL_PROPERTIES;//DEFAULT_PROPERTIES
		
		$request->ItemShape->AdditionalProperties = new SoapVar('
		   <ns1:AdditionalProperties>
		    <ns1:FieldURI FieldURI="calendar:OptionalAttendees"/>
		    <ns1:FieldURI FieldURI="calendar:RequiredAttendees"/>
		   </ns1:AdditionalProperties>', XSD_ANYXML); 
		 
		$request->CalendarView = new EWSType_CalendarViewType(); 
		
		$request->CalendarView->StartDate = $from_date->format('Y-m-d').'T00:01:34+03:00'; // an ISO8601 date e.g. 2012-06-12T15:18:34+03:00
		$request->CalendarView->EndDate = $upto_date->format('Y-m-d').'T23:55:23+03:00' ;// an ISO8601 date later than the above
		
		$request->ParentFolderIds = new EWSType_NonEmptyArrayOfBaseFolderIdsType();
		$request->ParentFolderIds->DistinguishedFolderId->Id = EWSType_DistinguishedFolderIdNameType::CALENDAR;
		
		$request->ParentFolderIds->DistinguishedFolderId->Mailbox = new EWSType_EmailAddressType();
		$request->ParentFolderIds->DistinguishedFolderId->Mailbox->EmailAddress = $username;
		
	
		$response = $cleint->FindItem($request);

Does anyone have experience getting php-ews to support this kind of functionality? I'm having trouble finding any usable code examples / documentation for this, so any help would be much appreciated.

**Comment by jamesiarmes** [comment]
I don't believe this is possible. Attempting to add the required or optional attendees as additional properties results in `ErrorInvalidPropertyForOperation`. You would have to do a `FindItem` request followed by a `GetItem` request in order to get the attendees.
